Rename us-benefits-policy.md to README.md

ContentBrewmaster opened the pull request:
Updating file name to handle new editing tool.

<!-- readthedocs-preview civicactions-handbook start -->
----
:books: Documentation preview :books:: https://civicactions-handbook--1152.org.readthedocs.build/en/1152/

<!-- readthedocs-preview civicactions-handbook end -->

ContentBrewmaster committed:
Rename us-benefits-policy.md to README.md

Updating file name to handle new editing tool.

ContentBrewmaster committed:
Updating links after file move or rename

ContentBrewmaster committed:
Merge branch 'master' into ContentBrewmaster-employee-benefits

dmundra committed:
Merge branch 'master' into ContentBrewmaster-employee-benefits

dmundra reviewed (approved):
Only a filename change! Thank you @ContentBrewmaster

ContentBrewmaster committed:
Update mkdocs.yml

Updated navigation file

dmundra commented on .config/mkdocs.yml:
```diff
@@ -27,11 +27,13 @@ nav:
       - company-policies/timesheets.md
       - company-policies/security.md
       - Employment policies (US employees):
+          - Employment: company-policies/README.md
```
@ContentBrewmaster looks like the company-policies/README.md change is not in this PR. Should this be in another PR?

ContentBrewmaster replied:
@dmundra I adjusted the file name.

dmundra committed:
Merge branch 'master' into ContentBrewmaster-employee-benefits

ContentBrewmaster committed:
Rename employment.md to README.md

Looking to make this page the landing page for the company policies section.

ContentBrewmaster committed:
Updating links after file move or rename

ContentBrewmaster committed:
Update mkdocs.yml

Looking to make US employment policy page the landing page for the policy section.

dmundra commented on .config/mkdocs.yml:
```diff
@@ -27,12 +27,14 @@ nav:
       - company-policies/security.md
       - Employment policies (US employees):
-          - company-policies/employment/us/employment.md
+          - Employment: company-policies/README.md
```
I think this link should be `Employment: company-policies/employment/us/README.md`? Right now it goes to a 404 when you try it here https://civicactions-handbook--1152.org.readthedocs.build/en/1152/company-policies/employment/us/

ContentBrewmaster replied:
@dmundra this should be ready to go?

dmundra replied:
Maybe not. I am not seeing the 'Employment' sub menu item in the sidebar. Maybe that has to go below the README? Like this:

```
          - company-policies/employment/us/README.md
          - Employment: company-policies/employment/us/README.md
```

dmundra reviewed (changes requested):
@ContentBrewmaster thanks for fixing the menu. I found one small issue.

ContentBrewmaster committed:
Merge branch 'master' into ContentBrewmaster-employee-benefits

ContentBrewmaster committed:
Update mkdocs.yml

Fixing link to new README in company policies section.
